paper-radio-button.html

```diff
@@ -71,7 +71,7 @@
       hostAttributes: {
         role: 'radio',
         'aria-checked': false,
-        tabindex: 0
+        tabindex: -1
```

Comment:
This is a problem if you don't include a button inside a `paper-radio-group` (which is a valid use case). In that case, I can never tab to unchecked buttons to check them. :(